src/test/java/org/hydev/mcpm/client/database/PluginMockFactory.java

```diff
@@ -42,6 +67,36 @@ public static PluginYml meta(String name, String version, String description) {
         );
     }
 
+    /**
+     * Creates a mock PluginYml object.
+     *
+     * @param name The name of the plugin.
+     * @param version The version string for the plugin.
+     * @param description The description for the plugin.
+     * @param commands Commands for the plugin.
+     * @return A PluginYml object.
+     */
+    public static PluginYml meta(String name, String version, String description,
```

Comment:
You should make the above `meta` function call this one. The above one is less specialized (just pass null to this guy!)